tests/integration_cases/test_membership_changes.rs

```diff
@@ -143,16 +136,16 @@ mod three_peers_add_voter {
         scenario.expect_read_and_dispatch_messages_from(&[1])?;
         scenario.assert_can_apply_transition_entry_at_index(
             &[2, 3],
-            2,
+            3,
             ConfChangeType::BeginMembershipChange,
         );
         scenario.assert_in_membership_change(&[1, 2, 3]);
 
         info!("Allowing new peers to catch up.");
-        scenario.expect_read_and_dispatch_messages_from(&[4, 1, 4])?;
+        scenario.expect_read_and_dispatch_messages_from(&[4, 1, 4, 1])?;
```

Comment:
Why there is an additional 1?

Reply:
Previously in 2-phase membership changes new added peers' `next_idx` is 1 but now it's leaders' `last_idx`. So new added followers will reject the first `msg append` from their leader, so their leader needs to send more a `msg append`.